File: Sort/BubbleSort/BubbleSort.php
function bubbleSort($arr) {
    for ($k = 0; $k < count($arr) - 1; $k++) {
        for ($i = 0; $i < count($arr) - 1 - $k; $i++) {
            if ($arr[$i] > $arr[$i + 1]) {
                $temp = $arr[$i];
                $arr[$i] = $arr[$i+1];
                $arr[$i+1] = $temp;
            }
        }
    }
}

$arr = array( 31, 6, 15, 1, -12, 10, 20 );

bubbleSort($arr);

print_r($arr);

File: Sort/BubbleSort/BubbleSort.java
import java.util.Arrays;

class BubbleSort {

	public static void bubbleSort(int arr[]) {
		for (int k = 0; k < arr.length - 1; k++) {
			for (int i = 0; i < arr.length - 1 - k; i++) {
				if (arr[i] > arr[i + 1]) {
					int temp = arr[i];
					arr[i] = arr[i+1];
					arr[i+1] = temp;
				}
			}
		}
	}

	public static void main(String[] args) {
		int arr[] = { 31, 6, 15, 1, -12, 10, 20 };

		bubbleSort(arr);

		System.out.println(Arrays.toString(arr));
	}
}